runtime/Stdlib_Int.resi

```diff
@@ -482,3 +482,33 @@ external shiftRightUnsigned: (int, int) => int = "%lsrint"
   without having to store or process it further.
 */
 external ignore: int => unit = "%ignore"
+
+module Ref: {
+  type t = Pervasives.ref<int>
```

Comment:
Cannot just use `ref` as this would be `Pervasives_mini.ref` then.
Maybe we can get rid of `Pervasives_mini` anyway, will look into that separately.

Reply:
Ok, might as well do it in this PR. 😄